webapp/app/components/MessageBox/MessageBox.component.js

```diff
@@ -0,0 +1,48 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+import { Wrapper, Field } from './MessageBox.styles';
+import { UserAvatar } from '../UserAvatar/UserAvatar.component';
+
+
+export class MessageBox extends PureComponent {
+  static propTypes = {
+    message: PropTypes.string,
+    onCreateMessage: PropTypes.func.isRequired,
+  };
+
+  state = {
+    message: '',
+  };
+
+  onChange = (event) => {
+    this.setState({
+      message: event.target.value,
+    });
+  };
+
+  onKeyUp = (event) => {
+    if (event.keyCode === 13) {
+      event.preventDefault();
+
+      this.props.onCreateMessage('Anonim', this.state.message);
+
+      this.setState({
+        message: '',
+      });
+    }
+  };
+
+  render = () => (
+    <Wrapper>
+      <UserAvatar />
+      <Field
+        value={this.state.message}
+        onChange={this.onChange}
```

Comment:
this.handleChange? ogolnie to raczej codestyle, ale wydaje mi sie to czytelniejsze, tak samo onKeyUp={this.handleKeyUp}. 'on' to wywolywany event a 'handle' to handler dla niego.